gcovr/gcov.py

```diff
@@ -376,6 +378,28 @@ def find_potential_working_directories_via_objdir(abs_filename, objdir, error):
     return []
 
 
+def check_gcov_option(gcov_cmd, env, option):
+
+    for help_option in ["--help", "--help-hidden"]:
+        gcov_process = subprocess.Popen(
+            gcov_cmd + [help_option],
+            env=env, stdout=subprocess.PIPE,
+            stderr=subprocess.PIPE,
+            universal_newlines=True,
+            encoding="utf-8"
+        )
+        out, _ = gcov_process.communicate()
+
+        if not gcov_process.returncode:
+            # gcov execution was not successful, help argument is not supported.
+            pass
```

Comment:
Gcov shall be called only one time with each option, caching the output in one global variable. One variable for both outputs can be used.